### PR TITLE
DevSpace: Add version 6.3.2

### PR DESCRIPTION
**Pull request by pauleHeisster**
Provide a new app manifest for [DevSpace](https://devspace.sh)

Relates to https://github.com/devspace-sh/devspace/issues/2629

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/devspace.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+    "version": "6.3.2",
+    "description": "DevSpace is an open-source developer tool for Kubernetes that lets you develop and deploy cloud-native software faster",
+    "homepage": "https://devspace.sh",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/devspace-sh/devspace/releases/download/v6.3.2/devspace-windows-amd64.exe",
```
Comment (aliesbelik) [inline bucket/devspace.json:8]:
```suggestion
            "url": "https://github.com/devspace-sh/devspace/releases/download/v6.3.2/devspace-windows-amd64.exe#/devspace.exe",
```

You can rename binary inline in the download URL.

File: bucket/devspace.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            "bin": [
+                "devspace-windows-amd64.exe",
+                [
+                    "devspace-windows-amd64.exe",
+                    "devspace"
+                ]
+            ]
+        },
+        "32bit": {
+            "url": "https://github.com/devspace-sh/devspace/releases/download/v6.3.2/devspace-windows-386.exe",
```
Comment (aliesbelik) [inline bucket/devspace.json:19]:
```suggestion
            "url": "https://github.com/devspace-sh/devspace/releases/download/v6.3.2/devspace-windows-386.exe#/devspace.exe",
```

File: bucket/devspace.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            "hash": "af143841ae2fa0c0a2e77ffa2ceb3d7a630458f17681d15d0e4fa8e922f259da",
+            "bin": [
+                "devspace-windows-386.exe",
+                [
+                    "devspace-windows-386.exe",
+                    "devspace"
+                ]
+            ]
+        }
+    },
```
Comment (aliesbelik) [inline bucket/devspace.json:29]:
```suggestion
    },
    "bin": "devspace.exe", 
```

File: bucket/devspace.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    "checkver": {
+        "github": "https://github.com/devspace-sh/devspace"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/devspace-sh/devspace/releases/download/v$version/devspace-windows-amd64.exe"
```
Comment (aliesbelik) [inline bucket/devspace.json:36]:
```suggestion
                "url": "https://github.com/devspace-sh/devspace/releases/download/v$version/devspace-windows-amd64.exe#/devspace.exe"
```

File: bucket/devspace.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+            },
+            "32bit": {
+                "url": "https://github.com/devspace-sh/devspace/releases/download/v$version/devspace-windows-386.exe"
```
Comment (aliesbelik) [inline bucket/devspace.json:39]:
```suggestion
                "url": "https://github.com/devspace-sh/devspace/releases/download/v$version/devspace-windows-386.exe#/devspace.exe"
```

File: bucket/devspace.json [hunk 6]
```diff
@@ -0,0 +1,46 @@
+{
+    "version": "6.3.2",
+    "description": "DevSpace is an open-source developer tool for Kubernetes that lets you develop and deploy cloud-native software faster",
+    "homepage": "https://devspace.sh",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/devspace-sh/devspace/releases/download/v6.3.2/devspace-windows-amd64.exe",
+            "hash": "72d89cd7f2c4b59a4ae282293423318eb5cac68a1bb48dcc50ff200f3ce826a6",
+            "bin": [
```
Comment (aliesbelik) [inline bucket/devspace.json:10]:
```suggestion
```

...so you can use single `binary` entry below instead.

File: bucket/devspace.json [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+                [
+                    "devspace-windows-amd64.exe",
+                    "devspace"
+                ]
+            ]
+        },
+        "32bit": {
+            "url": "https://github.com/devspace-sh/devspace/releases/download/v6.3.2/devspace-windows-386.exe",
+            "hash": "af143841ae2fa0c0a2e77ffa2ceb3d7a630458f17681d15d0e4fa8e922f259da",
+            "bin": [
```
Comment (aliesbelik) [inline bucket/devspace.json:21]:
```suggestion
```

**Review by aliesbelik — COMMENTED** [review]
You can use `#/` to rename binary inline in the download URL and make manifest more neat.

File: bucket/devspace.json (hunk 3 above)
Comment (pauleHeisster) [reply bucket/devspace.json:15]: done in https://github.com/ScoopInstaller/Main/pull/4737/commits/50e1e6f52444ec3a5892c7c3467bb3d5d0933724

File: bucket/devspace.json (hunk 6 above)
Comment (pauleHeisster) [reply bucket/devspace.json]: done in https://github.com/ScoopInstaller/Main/pull/4737/commits/50e1e6f52444ec3a5892c7c3467bb3d5d0933724

File: bucket/devspace.json (hunk 7 above)
Comment (pauleHeisster) [reply bucket/devspace.json]: done in https://github.com/ScoopInstaller/Main/pull/4737/commits/50e1e6f52444ec3a5892c7c3467bb3d5d0933724

**Comment by pauleHeisster** [comment]
> You can use `#/` to rename binary inline in the download URL and make manifest more neat.

Apparently i missed that in the documentation. Thanks for the hint!

**Pull request merged**